src/js/sniffer.js

```diff
@@ -26,6 +26,58 @@ const os = {
      **/
     get android() {
         return /android/i.test( ua );
+    },
+
```

Comment:
Maybe a bit nitpicky, but I'd like a separate browser object. We're trying to avoid sniffing and remove any sniffing code when we don't use it.

```
const browser = { get safari(){ return ua.includes('Safari')}
```